src/page/Home.js

```diff
@@ -0,0 +1,27 @@
+import { useFeaturedBanners } from '../utils/hooks/useFeaturedBanners';
+import Slider from '../widget/Slider';
+import CategoryGrid from '../widget/CategoryGrid';
+import ProductGrid from '../widget/ProductGrid';
+import categories from 'mocks/en-us/product-categories.json';
+import products from 'mocks/en-us/featured-products.json';
+
+const productsWithCategory = function () {
+    const categoryDictionary = new Map();
+    categories.results.map(({id, data: {name}})=> categoryDictionary.set(id, name));
+
+    return products.results.map((product) => { return { department: categoryDictionary.get(product.data.category.id), ...product }; });
+}();
+
+export default function Home() {
+    const { data: banners, isLoading: bannersLoading } = useFeaturedBanners();
+    
+    return (
+        <div className="home-wrapper">
+            { bannersLoading ? null : <Slider entries={banners?.results ?? []} /> }
```

Comment:
Yo personalmente haría que esa asignación de la propiedad `entries` en el Slider se resuelva antes en una constante, mejoraría legibilidad de la línea, porque ya de por sí tenemos conditional rendering.

Además podemos evitar el ternario si lo hacemos con el operador `&&`.

```suggestion
            const entries = banners?.results ?? [];
            { !bannersLoading && <Slider entries={entries} /> }
```